Fluid/Values/FluidValueExtensions.cs

```diff
@@ -95,7 +95,8 @@ public static bool TryGetDateTimeInput(this FluidValue input, TemplateContext co
             }
             else if (input.Type == FluidValues.Number)
             {
-                var dateTime = DateTimeOffset.FromUnixTimeSeconds((long)input.ToNumberValue());
+                var milliseconds = input.ToNumberValue() * 1000;
```

Comment:
Why did you change that?

Reply:
Because the millisecond precision was lost. If the span had `1.1` seconds, it was rounded down to `1` second.

Reply:
Issue is specific to this implementation, where we're treating the span as a number of seconds. But probably it doesn't make sense altogether, since now spans would be rendered as a number.